cpp/example-receiver.cc

```diff
@@ -0,0 +1,56 @@
+#include "zendar-api.h"
+#include "data.pb.h"
+
+int main(int argc, char* argv[]) {
+  if (argc < 2) {
+    printf("Usage: example-receiver URI");
+  }
+  zendar::ZendarReceiver rcv(argv[1]);
+  rcv.SubscribeImages(100);
+  rcv.SubscribeTracker(100);
+  rcv.SubscribeTracklog(100);
+  rcv.SubscribeLogs(100);
```

Comment:
Recommend checking return codes here and printing error messages for failures.

Reply:
Error messages are already printed for failures in the API